jupyterlab-dataops/Dockerfile

```diff
@@ -40,6 +36,21 @@ COPY resources/jupyter/ /etc/jupyter/
 # Expose port for Vertex AI compatibility
 EXPOSE 8080
 
-FROM base as bigquery
+# use default cloudbeaver in non-bigquery target
+FROM base AS default-stage
+COPY --from=dbeaver/cloudbeaver:22.0.5 /opt/cloudbeaver/ /opt/cloudbeaver/
+COPY resources/cloudbeaver/conf /opt/cloudbeaver/conf
+
+FROM base as bigquery-stage
 COPY requirements/requirements-bigquery.txt /opt/requirements/
-RUN pip install -r /opt/requirements/requirements-bigquery.txt
\ No newline at end of file
+RUN pip install -r /opt/requirements/requirements-bigquery.txt
+COPY --from=gcr.io/getindata-images-public/dataops-cloudbeaver:latest /opt/cloudbeaver/ /opt/cloudbeaver/
+COPY resources/cloudbeaver/conf /opt/cloudbeaver/conf
+ADD resources/cloudbeaver/conf/initial-data-sources-bigquery.conf /opt/cloudbeaver/conf/initial-data-sources.conf
+COPY resources/cloudbeaver/set_metadata.py /opt/cloudbeaver/set_metadata.py
+COPY resources/cloudbeaver/set_metadata_run_jupyter.sh /set_metadata_run_jupyter.sh
+RUN chmod +x /set_metadata_run_jupyter.sh
+CMD ["/set_metadata_run_jupyter.sh"]
+
+FROM ${PROVIDER}-stage as final
```

Comment:
What's the purpose of this FROM?

Reply:
That was supposed to be a way to make a multi-stage build with two independent targets (default/BQ).
I'm open to suggestion how you would see a multi-stage build to keep default & bigquery targets different.

Reply:
But you can easily create multistage build for many targets without this.

For reference check [this](https://github.com/getindata/dbt-images/blob/main/.github/workflows/release-images.yml#L13) In my matrix I'm storing aws and gcp and then in build stage, I'm using [this](https://github.com/getindata/dbt-images/blob/main/.github/workflows/release-images.yml#L50) with docker file looks like [this](https://github.com/getindata/dbt-images/blob/main/Dockerfile)
We have the possibility to keep many targets builds in a seperate way only adding additional requirements.

Reply:
Then it's something I must have mixed up lol. We'll do in similar way.